Android/BacktraceANRWatchdog.java

```diff
@@ -74,8 +65,10 @@ public BacktraceANRWatchdog(String gameObjectName, String methodName, int anrTim
     public void run() {
         Boolean reported = false;
         Log.d(LOG_TAG, "Starting ANR watchdog. Anr timeout: " + this.timeout);
+        if (Debug.isDebuggerConnected() || Debug.waitingForDebugger()) {
+            return;
```

Comment:
Maybe we should keep a log message here?

Reply:
done, thx.